src/langsmith/trace-with-langchain.mdx

```diff
@@ -292,9 +288,20 @@ await chain.invoke({ input: "What is the meaning of life?" }, {runName: "MyCusto
 
 </CodeGroup>
 
+<Note>
+The `run_name` parameter only changes the name of the runnable you invoke (e.g., a chain, function). It does not rename the nested run automatically created when you invoke an LLM object like `ChatOpenAI` (`gpt-4o-mini`).
+
+In the example, the enclosing run will appear in LangSmith as `MyCustomChain`, while the nested LLM run still shows the model’s default name.
```

Comment:
nit but i don't think these all need to be line breaks. can collapse into a paragraph until the bullets